sonar-application/src/main/assembly/conf/sonar.properties

```diff
@@ -213,9 +213,17 @@
 #--------------------------------------------------------------------------------------------------
 # LOGGING
 
-# Level of logs. Supported values are INFO(default), DEBUG and TRACE (DEBUG + SQL + ES requests)
+# Global level of logs (applies to App, Web and CE processes).
+# Supported values are INFO (default), DEBUG and TRACE (on web server: TRACE = DEBUG + SQL + ES requests)
```

Comment:
TRACE level should not have a specific behaviour in web server. It should enable all TRACE logs, not only SQL and ES.